### PR TITLE
feat: Implement remove and import command

### PR DESCRIPTION
**Pull request by benwoo1110**
Implement import and remove command

### CODE REVIEW

File: src/main/java/com/onarandombox/MultiverseCore/commands/ImportCommand.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    @CommandPermission("multiverse.core.import")
+    @CommandCompletion("@mvworlds:scope=potential  @flags:groupName=mvimport")
+    @Syntax("<name> <env> --generator [generator[:id]] --adjust-spawn")
+    @Description("Imports a existing world folder.")
+    public void onImportCommand(BukkitCommandIssuer issuer,
+
+                                @Conditions("validWorldName:scope=new")
+                                @Syntax("<name>")
+                                @Description("Folder name of the world.")
+                                String worldName,
```
Comment (zax71) [inline src/main/java/com/onarandombox/MultiverseCore/commands/ImportCommand.java:60]: `folderName` makes more sense to me

File: src/main/java/com/onarandombox/MultiverseCore/commands/ImportCommand.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+                                World.Environment environment,
+
+                                @Optional
+                                @Syntax("--generator [generator[:id]] --adjust-spawn")
+                                @Description("Other world settings. See: http://gg.gg/nn8c2")
+                                String[] flags) {
+
+        ParsedCommandFlags parsedFlags = parseFlags(flags);
+
+        issuer.sendMessage(String.format("Starting import of world '%s'...", worldName));
```
Comment (zax71) [inline src/main/java/com/onarandombox/MultiverseCore/commands/ImportCommand.java:73]: No need to say `world <worldName>`, would be better as
```java
issuer.sendMessage(ChatColor.GREEN + "Starting import of " + ChatColor.GOLD + worldName);
```

**Review by zax71 — APPROVED** [review]
Again, just UX stuff

File: src/main/java/com/onarandombox/MultiverseCore/commands/ImportCommand.java (hunk 1 above)
Comment (benwoo1110) [reply src/main/java/com/onarandombox/MultiverseCore/commands/ImportCommand.java:60]: I mean the folder name is the world name tbh, but ig can change to differentiate from create command

File: src/main/java/com/onarandombox/MultiverseCore/commands/ImportCommand.java (hunk 2 above)
Comment (benwoo1110) [reply src/main/java/com/onarandombox/MultiverseCore/commands/ImportCommand.java:73]: hmm yea, tbh we need to standardise our output formatting colour and stuff

File: src/main/java/com/onarandombox/MultiverseCore/commands/ImportCommand.java (hunk 1 above)
Comment (dumptruckman) [reply src/main/java/com/onarandombox/MultiverseCore/commands/ImportCommand.java:60]: I dunno, I'm not sure that is significant enough to warrant changing this from prior expectations of general terms we use.

Comment (benwoo1110) [reply src/main/java/com/onarandombox/MultiverseCore/commands/ImportCommand.java:60]: yea, users wont see our variables anyways

Comment (dumptruckman) [reply src/main/java/com/onarandombox/MultiverseCore/commands/ImportCommand.java:60]: Maybe the description would read better like "Name of the world folder".

File: src/main/java/com/onarandombox/MultiverseCore/commands/ImportCommand.java (hunk 2 above)
Comment (dumptruckman) [reply src/main/java/com/onarandombox/MultiverseCore/commands/ImportCommand.java:73]: I have given thought to that problem before but had trouble coming up with a good solution. It's hard to come up with a concise list of "categories" to assign colors to.

Comment (benwoo1110) [reply src/main/java/com/onarandombox/MultiverseCore/commands/ImportCommand.java:73]: yea, i see other plugins most colours are hard coded also

Comment (benwoo1110) [reply src/main/java/com/onarandombox/MultiverseCore/commands/ImportCommand.java:73]: Currently not too concerned about how fancy the output is, will be moved to locales anyways

**Pull request merged**